code/modules/reagents/Chemistry-Recipes/Pyrotechnics.dm

```diff
@@ -34,6 +34,41 @@
 	e.start()
 	holder.clear_reagents()
 
+/datum/chemical_reaction/reagent_explosion/potassium_explosion/holyboom
+	name = "Holy Explosion"
+	id = "holyboom"
+	required_reagents = list("holywater" = 1, "potassium" = 1)
+
+/datum/chemical_reaction/potassium_explosion/holyboom/on_reaction(datum/reagents/holder, created_volume)
+	var/atom/H = holder.my_atom
+	var/location = get_turf(holder.my_atom)
+	var/datum/effect_system/reagents_explosion/e = new()
+	world<<"created [created_volume] holyboom"
+	if(created_volume >= 150)
```

Comment:
will fix when i get home, the /tg/ version has it as a lategame item you need bluespace beakers/grenades for.